app/services/intl.js

```diff
@@ -4,9 +4,9 @@
  */
 
 import Ember from 'ember';
-import Locale from 'ember-intl/models/locale';
-import createFormatCache from 'ember-intl/format-cache/memoizer';
 import { IntlRelativeFormat, IntlMessageFormat } from 'ember-intl/utils/data';
+import createFormatCache from 'ember-intl/format-cache/memoizer';
```

Comment:
Ya, there was a bug in browserify that was in a more recent build.  First I need to fix ember-browserify in order to use this.  Plan on doing this though.